### PR TITLE
[reapop]: add definitions for module

### PR DESCRIPTION
**Pull request by Barrokgl**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, and `strictNullChecks` set to `true`.

### CODE REVIEW

File: types/reapop/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,165 @@
+// Type definitions for reapop 1.1
+// Project: https://github.com/LouisBarranqueiro/reapop#readme
+// Definitions by: Barrokgl <https://github.com/Barrokgl>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1.1
```
Comment (rbuckton) [inline types/reapop/index.d.ts]: The version specified here is not correct. At a minimum to support React, you need to use `// TypeScript Version: 2.3`

**Review by rbuckton — CHANGES_REQUESTED** [review]
I restarted the build as it failed due to an npm issue. The updated build result now indicates issues with this definition.

File: types/reapop/index.d.ts (hunk 1 above)
Comment (Barrokgl) [inline types/reapop/index.d.ts]: updated version

**Comment by Barrokgl** [comment]
@rbuckton with  updated version test are successful

**Pull request merged**